amazon-kinesis-client-multilang/src/main/java/software/amazon/kinesis/multilang/config/KinesisClientLibConfigurator.java

```diff
@@ -19,12 +19,14 @@
 import java.lang.reflect.InvocationTargetException;
 import java.util.Properties;
 
+import com.amazonaws.arn.Arn;
 import org.apache.commons.beanutils.BeanUtilsBean;
 import org.apache.commons.beanutils.ConvertUtilsBean;
 
 import lombok.extern.slf4j.Slf4j;
 import org.apache.commons.lang3.StringUtils;
 import org.apache.commons.lang3.Validate;
+import software.amazon.awssdk.regions.Region;
```

Comment:
The `com.amazonaws` is from SDK v1 while `software.amazon` is from SDK v2. Looking at the `pom.xml`, the multilang KCL has dependencies on both SDK v1 and V2...

There appears to be more usages of SDK v2 ([search ref](https://github.com/search?q=repo%3Aawslabs%2Famazon-kinesis-client+path%3A%2F%5Eamazon-kinesis-client-multilang%5C%2Fsrc%5C%2Fmain%5C%2F%2F+software.amazon.awssdk&type=code)) than v1 ([search ref](https://github.com/search?q=repo%3Aawslabs%2Famazon-kinesis-client+path%3A%2F%5Eamazon-kinesis-client-multilang%5C%2Fsrc%5C%2Fmain%5C%2F%2F+com.amazonaws&type=code)). I think the best bet is to use SDK v2 where possible

Reply:
Good callout, will get that updated.